server/src/main/java/org/apache/iotdb/db/service/IoTDB.java

```diff
@@ -109,7 +109,7 @@ public void active() {
 
     // set recover config, avoid creating deleted time series when recovering wal
     boolean prevIsAutoCreateSchemaEnabled = config.isAutoCreateSchemaEnabled();
-    config.setAutoCreateSchemaEnabled(false);
+    config.setAutoCreateSchemaEnabled(true);
```

Comment:
why change the mode? We need to disable auto create when recover.

Reply:
Okay, got it.The state was changed to find the error.